resteasy-client-microprofile/src/main/java/org/jboss/resteasy/microprofile/client/header/ClientHeadersRequestFilter.java

```diff
@@ -92,8 +90,6 @@ public void filter(ClientRequestContext requestContext) {
                 .forEach(
                         (key, values) -> requestContext.getHeaders().put(key, castToListOfObjects(values))
                 );
-
-        ResteasyProviderFactory.getContextDataMap().put(HttpHeaders.class, new HttpHeadersContextProvider(requestContext));
```

Comment:
There used to be a test in MP Rest Client TCK that required it but the test is removed now.

Reply:
Great one less thing in the map there.